src/components/HOCs/WithSearchResults/WithSearchResults.js

```diff
@@ -113,4 +132,4 @@ export default (WrappedComponent, searchName, itemsProp, outputProp, searchFunct
     WithSearchResults(WrappedComponent, searchName, itemsProp, outputProp),
     searchName,
     searchFunction
-  )
+  )
```

Comment:
please revert this file and create a new HOC specifically for metrics.  too many conditions just for admin and too easy to create regressions this way.